.wakeroot

```diff
@@ -1,2 +1,4 @@
 {
+    "log_header": "",
+    "log_header_extra_width": 0
```

Comment:
any specific reason we aren't using the default here?

Reply:
Because then it would clutter the stdout with info

Reply:
🤷‍♂️ fair, I disagree but I suppose that is why it's configurable